data/json/items/comestibles/other.json

```
@@ -527,6 +527,50 @@
     "fun": 1,
     "vitamins": [ [ "calcium", 8 ], [ "veggy_allergen", 1 ] ]
   },
+  {
+    "type": "ITEM",
+    "subtypes": [ "COMESTIBLE" ],
+    "id": "liquid_soy_lecithin",
+    "name": { "str_sp": "liquid soy lecithin" },
+    "//": "Details from https://shop.chefrubber.com/images/items/pdf/nutritional/Additives/Lecithin%20Liquid-Soy.pdf",
+    "weight": "10 g",
+    "color": "light_green_yellow",
+    "spoils_in": "1825 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 73,
+    "description": "Liquid soy lecithin is a golden liquid used as an emulsifier and smoothing agent in food manufacture.",
+    "price": "1 cent",
+    "price_postapoc": "1 cent",
+    "fun": -4,
+    "phase": "liquid",
+    "material": [ "veggy" ],
+    "volume": "10 ml",
+    "flags": [ "NUTRIENT_OVERRIDE" ],
+    "vitamins": [ [ "calcium", 10 ], [ "iron", 17 ], [ "veggy_allergen", 1 ] ]
+  },
+  {
+    "type": "ITEM",
+    "subtypes": [ "COMESTIBLE" ],
+    "id": "powder_soy_lecithin",
+    "name": { "str_sp": "granule soy lecithin" },
+    "//": "Details from https://www.nutritionix.com/i/now/lecithin-granules/589ac28aa703481c6ba10b80",
+    "weight": "10 g",
+    "color": "white",
+    "spoils_in": "730 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 70,
+    "description": "Granule soy lecithin is a white powder used as an emulsifier and smoothing agent in food manufacture.",
+    "price": "1 cent",
+    "price_postapoc": "1 cent",
+    "fun": -14,
+    "phase": "powder",
```

Invalid enum string 'powder' for 'phase_id'
```suggestion
    "phase": "solid",
```